store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -168,6 +164,17 @@ func (h *rpcHandler) buildDAGExecutor(req *coprocessor.Request) (*dagContext, ex
 	return ctx, e, dagReq, err
 }
 
+// constructTimeZone constructs timezone by name first. When the timezone name
+// is set and is not equal to "UTC", the daylight saving time problem must be
+// considered. Otherwise the timezone is constructed directly from the offset.
```

Comment:
It is not exactly `offset` but time zone offset in seconds. We're better to comment it. 
'